packages/expo-image/src/Image.tsx

```diff
@@ -74,3 +74,43 @@ export class Image extends React.PureComponent<ImageProps> {
     );
   }
 }
+
+export class ImageBackground extends React.PureComponent<ImageBackgroundProps> {
+  /**
+   * Preloads images at the given urls that can be later used in the image view.
+   * Preloaded images are always cached on the disk, so make sure to use
+   * `disk` (default) or `memory-disk` cache policy.
+   */
+  static prefetch(urls: string | string[]): void {
```

Comment:
I think there is no need to copy these functions as they would then appear twice in the documentation. Let's keep static methods only in the `Image` component.

Reply:
No problem, that makes sense. Just thought it would stop users needing to import both `Image` and `ImageBackground` to use them.